### ISSUE TITLE
Add quiet mode

### ISSUE THREAD

**Issue by ebbishop** [protomaps/go-pmtiles#117]
**pmtiles version:** 1.11.0, 926183a1, built at 2023-10-31T00:41:14Z
**OS:** MacOS 11.7 and Ubuntu 22.04

It would be great to have a quiet mode available, especially when running `pmtiles` as part of a CI/CD pipeline. Converting mbtiles to pmtiles in our pipeline currently produces ~4,000 lines of progress bar logs. This doesn't add anything of value in this context, and it would be helpful to have a way to turn that off.

**Comment by bdon** [comment]
This would a good addition. I'd prefer keeping progress the default and adding `--quiet` instead of making quiet the default and adding `--progress`. What program is earlier in your pipeline that creates MBTiles?

**Comment by ebbishop** [comment]
Agreed, it makes sense to have `--quiet` as an opt-in feature. 

We're using `tippecanoe` to build mbtiles from a PostGIS db, then converting to pmtiles.

**Comment by bdon** [comment]
If you are using the maintained version of tippecanoe https://github.com/felt/tippecanoe you can generate PMTiles directly and don't need to use go-pmtiles at all.

**Comment by bdon** [comment]
* move to go 1.21 and structured logging https://pkg.go.dev/log/slog to aid this

**Comment by mem48** [comment]
I'm not sure if this is related but I'm experiencing really slow performance on converting `mbtiles` to `pmtiles`.

My dataset is a 158 GB `.mbtiles` containing 2m resolution raster data saved as `webp` images produced from `rio rgbify`.

I'm currently on Pass 2: writing tiles with the progress bar saying:

`11% | (405094/3638545, 54 it/min) [57h56m55s:989h46m23s]`

I know it is a large dataset, but surely it shouldn't take 40 days on a high-spec desktop. The PC doesn't appear to be working hard (10% CPU, 8% Memory, 1% Disk)

**Comment by jackiescanlon** [comment]
Wondering if this might be supported soon? My use case is quieting the `pmtiles serve` command.

**Comment by bdon** [comment]
The issue above was regarding `pmtiles convert`, do you want `pmtiles serve` to output nothing at all? In most cases logging one line per served request with an HTTP status code is useful.

**Comment by jackiescanlon** [comment]
Yeah, a `--quiet` mode for `pmtiles serve` would be helpful. It is helpful to have the output for initial debugging but otherwise we'd love to quiet it down.

**Comment by bdon** [comment]
Piping the output to `/dev/null` could be one solution, unless you want to see only initialization messages, error messages etc and not info messages.

**Comment by jackiescanlon** [comment]
Yeah exactly, it'd still be good to see the other messages, just not the stream of served request messages.
